images/build/php/prereqs/installPrereqs.sh

```diff
@@ -33,7 +33,10 @@ apt-get update \
     --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
 ##<argon2>##
-sed -e 's/stretch/buster/g' /etc/apt/sources.list > /etc/apt/sources.list.d/buster.list;
+sed -e 's/# deb http:\/\/archive.debian.org\/debian-security stretch/deb http:\/\/security.debian.org\/debian-security stretch/g' \
```

Comment:
Does the commented out line `# deb http://archive.debian.org/debian-security stretch` exist at this point for it to replace back with the security version? I don't see that link getting created above

Reply:
this reverting step is mainly for populating the `buster.list`. It'll revert the changes to what the `sources.list` originally looks like so that the `buster.list` could be created correctly. That line will be changed to `deb http://security.debian.org/debian-security buster`.

Reply:
oh, I see. I think that this line should just instead revert the `stretch-updates` line, e.g something like:

`s/# deb http:\/\/deb.debian.org\/debian stretch-updates/deb http:\/\/deb.debian.org\/debian stretch-updates/g`

Reply:
since we have 2 lines in this block that are updating `debian-security` source

Reply:
and original changes are to:
```
# deb http://deb.debian.org/debian stretch-updates             (comment this out, as stretch-updates didn't seem to exist)
deb http://archive.debian.org/debian-security stretch          (archive source)
deb http://archive.debian.org/debian stretch                   (archive source)
```

Reply:
makes sense! Just pushed a new change removing the duplicate line.